checker/src/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -159,6 +159,17 @@ protected boolean isInitializationAnnotation(AnnotationMirror anno) {
      */
     public abstract AnnotationMirror getFieldInvariantAnnotation();
 
+    /**
+     * Returns whether or not field has the invariant annotation.
```

Comment:
Replacing `field` with `{@code field}` would make it clearer that the parameter is meant here.